dts/arm/ti/lm3s6965.dtsi

```diff
@@ -56,6 +56,17 @@
 			status = "disabled";
 			label = "UART_2";
 		};
+
```

Comment:
Add this in a separate patch (as well for the board enabling spi, the fixup etc.. )